app/assets/stylesheets/pageflow/editor/dialogs.scss

```diff
@@ -5,6 +5,7 @@
   width: 100%;
   height: 100%;
   z-index: 5;
+  background-color: transparent;
```

Comment:
It is just an artifact of trying to reproduce exactly the styles of the text tracks dialog. Testing revealed that it is not needed. I just thought it was